velox/type/Type.h

```diff
@@ -929,6 +1034,8 @@ std::shared_ptr<const TimestampType> TIMESTAMP();
 
 std::shared_ptr<const DateType> DATE();
 
+std::shared_ptr<const DecimalType> DECIMAL();
```

Comment:
There is no default Decimal type, is there? E.g. one always need to pass precision and scale. Let's remove this.